platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/AbstractStsRealm.java

```diff
@@ -237,24 +244,32 @@ protected SecurityToken requestSecurityToken(Object authToken) {
      * @param securityToken The token being renewed.
      * @return security token (SAML assertion)
      */
-    protected SecurityToken renewSecurityToken(SecurityToken securityToken) {
+    protected SecurityToken renewSecurityToken(final SecurityToken securityToken) {
```

Comment:
A little cleaner version of this method is here: https://gist.github.com/coyotesqrl/f4ffa0c585fabd6500241c91ef2dc54b

Either works, but this takes advantage of the cache's functionality a bit more.